cves/kernel/CVE-2014-4943.yml

```diff
@@ -184,8 +189,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    Nothing referenced any specifications.  The bug report, because it was posted after the patch,
```

Comment:
I appreciated your effort but you might want to change the beginning with the start of nothing, making this mean nothing is absolute, where some specification may exist, just be nested somewhere.